### ISSUE TITLE
`make docs-linkcheck` failing for various Tails URLs

### ISSUE THREAD

**Issue by legoktm** [freedomofpress/securedrop-docs#298]
## Expected behavior

`make docs-linkcheck` passes

## Actual behavior

```
/home/user/github/freedomofpress/securedrop-docs/docs/servers.rst.rst:335:broken link: https://tails.boum.org/doc/first_steps/persistence/configure/index.en.html#index3h2 (404 Client Error: Not Found for url: https://tails.boum.org/doc/first_steps/persistence/configure/index.en.html/)
```

then I fixed that one (should go to https://tails.boum.org/doc/first_steps/persistence/index.en.html#index12h2), ran it again:

```
/home/user/github/freedomofpress/securedrop-docs/docs/onboarding_admins.rst.rst:28:broken link: https://tails.boum.org/doc/first_steps/persistence/use/index.en.html (404 Client Error: Not Found for url: https://tails.boum.org/doc/first_steps/persistence/use/index.en.html/)
```

and then I assumed more links would be broken besides these two and didn't keep looking for the new URLs. I also didn't yet look to see if someone has filed a bug with Tails about the broken links.

**Comment by conorsch** [comment]
Thanks for filing. I checked, and we _are_ running the linkcheck nightly in CI, and it failed last night (of course): https://app.circleci.com/pipelines/github/freedomofpress/securedrop-docs/1075/workflows/4bf3c2a1-5f22-4b61-9824-8a252add40ff/jobs/1052 but it's easy for those checks to get buried in scrollback.

**Comment by eloquence** [comment]
This was resolved via #296; linkcheck will fail until #299 is also merged.

**Comment by legoktm** [comment]
Oops, missed the PR. Thanks!
